### PR TITLE
Minor editorial and spelling fixes + parts of issue #242

### PR DESCRIPTION
**Pull request by KSchopmeyer**
Fixes a number of mispellings and minor editorial issues.
This covers several files but is just minor changes.

In reviewing issue #242, I concluded that this is going to be more than one PR.  Therefore I added some of the easy parts to this pr.  NOte that it is in a second commit.

### CODE REVIEW

File: docs/appendix.rst [hunk 1]
```diff
@@ -89,7 +90,8 @@ This documentation uses a few special terms to refer to Python types:
    WBEM-URI
       The wbem-uri is a standardized text form for CIM instance names. It is
       documented in DMTF :term:`DSP0207`. Pywbemcli uses the untyped WBEM URI
-      as the format for instance names in cli input parameters
+
```
Comment (andy-maier) [inline docs/appendix.rst]: An extra line slipped in.

File: docs/introduction.rst [hunk 2]
```diff
@@ -79,12 +79,12 @@ Supported environments
...
-The pywbemcli module of the pywbemtools package is supported in these
-environments:
+The pywbemtools package is supported in these environments:
 
-* Operating systems: Linux, Windows, macOS
+* Operating systems: Linux, Windows (native and Unix-like environments
+  (ex. Cygwin), OS-X
 * Python versions: 2.7, 3.4, and greater
-* pywbem 0.14.4 or greater
+* pywbem
```
Comment (andy-maier) [inline docs/introduction.rst]: We can remove the "pywbem" item entirely. If we don't specify the minimum version here, then there is nothing on it that is worth mentioning in a section about supported environments.

File: docs/appendix.rst (hunk 1 above)
Comment (KSchopmeyer) [reply docs/appendix.rst]: Fixed

File: docs/introduction.rst (hunk 2 above)
Comment (KSchopmeyer) [reply docs/introduction.rst]: Agreed. Removed

**Comment by KSchopmeyer** [comment]
Comments corrected and rebased.

**Comment by andy-maier** [comment]
Approved and squashing during merge.

**Pull request merged**